src/features/surveys/repos/SurveysRepo.ts

```diff
@@ -13,17 +24,45 @@ import {
   surveyUpdated,
 } from '../store';
 import { IFuture, PromiseFuture, RemoteItemFuture } from 'core/caching/futures';
-import {
-  ZetkinSurvey,
-  ZetkinSurveyElement,
-  ZetkinSurveyExtended,
-  ZetkinSurveySubmission,
-} from 'utils/types/zetkin';
+
+export type ZetkinSurveyElementPostBody =
+  | Partial<Omit<ZetkinSurveyTextElement, 'id'>>
+  | ZetkinSurveyTextQuestionElementPostBody
+  | ZetkinSurveyOptionsQuestionElementPostBody;
+
+type ZetkinTextQuestionPostBody = Omit<ZetkinTextQuestion, 'required'>;
```

Comment:
Because this type is not used anywhere else, I would suggest just inlining it on line 36.